admin/src/locales/de.json

```diff
@@ -126,11 +127,26 @@
       "title": "Gemeinwohl-Beitrag löschen!",
       "yes": "Ja, Beitrag löschen!"
     },
+    "deleteUser": {
```

Comment:
We should think about grouping this into
```
modals: {
  modalA:{
    question: ...
  },
  modalB: {..}
}
```
no?

Reply:
It is grouped under `overlay` in the locales.